egs2/coraal/asr1/README.md

```diff
@@ -0,0 +1,85 @@
+# CORAAL (Corpus of Regional African American Language)
```

Comment:
You can combine readme and results into a single file as the content are highly related.

Reply:
Sounds good!